### PR TITLE
separate out the style guide

### PR DESCRIPTION
**Pull request by bengl**

### CODE REVIEW

File: STYLE-GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Style Guide
+
+* Documents are written in markdown files.
+* Those files should be written in **`lowercase-with-dashes.md`.**
+  * Underscores in filenames are allowed only when they are present in the
+    topic the document will describe (e.g., `child_process`.)
+  * Filenames should be **lowercase**.
+  * Some files, such as top-level markdown files, are exceptions.
```
Comment (bengl) [inline STYLE-GUIDE.md:8]: Added this line.

File: STYLE-GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  * Inside the wrapping element if the wrapping element contains a complete
+    clause — a subject, verb, and an object.
+  * Outside of the wrapping element if the wrapping element contains only a
+    fragment of a clause.
+* Place end-of-sentence punctuation inside wrapping elements — periods go
+  inside parentheses and quotes, not after.
+* Documents must start with a level-one heading. An example document will be
+  linked here eventually.
+* Prefer affixing links to inlining links — prefer `[a link][]` to
+  `[a link](http://example.com)`.
```
Comment (bengl) [inline STYLE-GUIDE.md:38]: Changed this to an actual URL.

**Comment by chrisdickinson** [comment]
:100: This LGTM!

**Comment by distracteddev** [comment]
+1 LGTM

**Pull request merged**